contrib/soju/template.py

```diff
@@ -0,0 +1,79 @@
+pkgname = "soju"
+pkgver = "0.7.0"
+pkgrel = 0
+build_style = "makefile"
+hostmakedepends = ["go", "scdoc"]
+makedepends = ["sqlite-devel"]
+depends = ["ca-certificates"]
+pkgdesc = "IRC bouncer"
+maintainer = "ttyyls <contact@behri.org>"
+license = "AGPL-3.0-or-later"
+url = "https://soju.im"
+source = f"https://git.sr.ht/~emersion/soju/archive/v{pkgver}.tar.gz"
+sha256 = "089b900ccd7fa009b7ae50b9c2b9e77099b50b2499422649c3b27aff7d62d109"
+env = {"GOFLAGS": "-tags=libsqlite3"}
+file_xattrs = {
+    "usr/bin/soju": {
+        "security.capability": "cap_net_bind_service,cap_net_admin+ep",
```

Comment:
this absolutely does not need cap_net_admin

Reply:
it also makes more sense to use capsh in the service file than to set the capability here, since only the service might need it